decoders/connector/lansitec/contact-tracing-badge/v1.0.0/payload.js

```diff
@@ -6,68 +6,73 @@
 function Decode(fPort, bytes, variables) {

```

Comment:
This decoder must have a unit test, you can use the example I posted in the comments above, but if you can add the unit test for the payload parser it will be great.